source/patterns/@aws-solutions-constructs/aws-s3-stepfunctions/test/integ.s3-stepfunctions-no-argument.ts

```diff
@@ -32,7 +34,16 @@ const props: S3ToStepfunctionsProps = {
   logGroupProps: {
     removalPolicy: RemovalPolicy.DESTROY
   },
+  logS3AccessLogs: false,
+  deployCloudTrail: false
```

Comment:
If we left this set to true (or default) on one test, I'm guessing it leaves behind a bucket?

Reply:
Yes, that is correct

Reply:
Do we have a unit test that sets deployCloudTrail to true so we test it somewhere?

Reply:
No, I turned it off so it doesn't leave any CloudTrail logging buckets for all tests. Did you want me to leave it on for a test?